lib/core/customer.js

```diff
@@ -401,14 +404,44 @@ class CustomerService {
         return response.data;
     }
 
+    /**
+   * Create a single Secondary Customer
+   *
+   * Creates a new instance of a customer.
+   * @param {string} externalUid - A Customer identifier supplied by the Client, unique among the collection of all Client Customers - optional
```

Comment:
```suggestion
   * @param {string} [externalUid] - A Customer identifier supplied by the Client, unique among the collection of all Client Customers 
```

We can signify an optional param like this
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values